src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -57,14 +42,9 @@ Array [
     className="indicator-breakdown"
   >
     Progress
-    : 
-    19
-     
-    of
-     
-    22
+    :
      
-    rooms
+    19 of 22 room
```

Comment:
It seems like we lost the plural form of room @cKellyDesign

Reply:
changed to `room(s)`